solidity/ts/testsuite/simulator/types/ethSimulateTypes.ts

```diff
@@ -100,7 +109,7 @@ const EthSimulateCallResultFailure = funtypes.ReadonlyObject({
 	  gasUsed: EthereumQuantitySmall,
 	  error: funtypes.ReadonlyObject({
 		  code: funtypes.Number,
-		  message: funtypes.String,
+		  message: funtypes.String
```

Comment:
I prefer leaving the trailing comma when doing multi-line objects or function calls like this.  That way if you add a new one, you don't get a change on two lines.  Also makes refactoring (like re-ordering) easier as you can just alt-up/down.